src/configureStore.js

```diff
@@ -21,7 +21,7 @@ function configureStore(history) {
   const enhancer = composeEnhancers(
     applyMiddleware(...middlewares),
   );
-  const store = createStore(reducer, enhancer);
+  const store = createStore(reducer, initialState, enhancer);
```

Comment:
The store now accepts a initialState, which allows to hydrate the store with existing data